packages/styled-components/src/models/StyleSheetManager.js

```diff
@@ -16,25 +17,32 @@ export const StyleSheetConsumer = StyleSheetContext.Consumer;
 export const masterSheet: StyleSheet = new StyleSheet(false);
 
 export function useStyleSheet(): StyleSheet {
-  const fromContext = useContext(StyleSheetContext);
-  return fromContext !== undefined ? fromContext : masterSheet;
+  const sheet = useContext(StyleSheetContext);
+  return sheet !== undefined ? sheet : masterSheet;
 }
 
 function useStyleSheetProvider(sheet?: StyleSheet, target?: HTMLElement) {
-  return useMemo(() => {
-    if (sheet) {
-      return sheet;
-    } else if (target) {
-      return new StyleSheet(false, target);
-    } else {
-      throw new StyledError(4);
-    }
-  }, [sheet, target]);
+  return useMemo(
+    () => {
+      if (sheet) {
+        return sheet;
+      } else if (target) {
+        return new StyleSheet(false, target);
+      } else {
+        return masterSheet;
```

Comment:
when using `StyleSheetManager` if `sheet` isn't passed it should just fall back to masterSheet